src/components/favorite-card/favorite-card.html

```diff
@@ -0,0 +1,225 @@
+        fill="#660C00"
+      />
+      <path
+        d="M15.9881 23.8296C15.9881 23.2945 16.2471 22.4995 16.6465 21.9452C16.3768 21.571 16.0429 21.3063 15.6816 21.3063C14.7851 21.3063 14.0583 22.9331 14.0583 23.8294C14.0583 24.726 14.7852 25.4526 15.6816 25.4526C16.0435 25.4526 16.3766 25.3327 16.6465 25.1325C16.2478 24.837 15.9881 24.3643 15.9881 23.8296Z"
+        fill="#660C00"
+      />
+      <path
+        d="M25.2634 23.8296C25.2634 24.7261 24.5366 25.4528 23.6402 25.4528C22.7436 25.4528 22.017 24.726 22.017 23.8296C22.017 22.9332 22.7437 21.3065 23.6402 21.3065C24.5366 21.3065 25.2634 22.9331 25.2634 23.8296Z"
+        fill="#660C00"
+      />
+      <path
+        d="M23.9466 23.8296C23.9466 23.2945 24.2056 22.4995 24.605 21.9452C24.3353 21.571 24.0014 21.3063 23.6401 21.3063C22.7436 21.3063 22.0168 22.9331 22.0168 23.8294C22.0168 24.726 22.7437 25.4526 23.6401 25.4526C24.002 25.4526 24.3351 25.3327 24.605 25.1325C24.2062 24.837 23.9466 24.3643 23.9466 23.8296Z"
+        fill="#660C00"
+      />
+      <path
+        d="M21.2841 31.7049C21.2841 32.6014 20.5573 33.3281 19.6609 33.3281C18.7644 33.3281 18.0377 32.6013 18.0377 31.7049C18.0377 30.8084 18.7645 29.1818 19.6609 29.1818C20.5573 29.1817 21.2841 30.8084 21.2841 31.7049Z"
+        fill="#660C00"
+      />
+      <path
+        d="M19.9674 31.7049C19.9674 31.1699 20.2264 30.3748 20.6258 29.8205C20.356 29.4463 20.0222 29.1816 19.6609 29.1816C18.7643 29.1816 18.0376 30.8083 18.0376 31.7047C18.0376 32.6013 18.7644 33.3279 19.6609 33.3279C20.0228 33.3279 20.3559 33.208 20.6258 33.0078C20.2269 32.7123 19.9674 32.2396 19.9674 31.7049Z"
+        fill="#660C00"
+      />
+      <path
+        d="M13.3257 31.7049C13.3257 32.6014 12.599 33.3281 11.7025 33.3281C10.8061 33.3281 10.0793 32.6013 10.0793 31.7049C10.0793 30.8084 10.8061 29.1818 11.7025 29.1818C12.599 29.1818 13.3257 30.8084 13.3257 31.7049Z"
+        fill="#660C00"
+      />
+      <path
+        d="M12.009 31.7049C12.009 31.1699 12.268 30.3748 12.6674 29.8205C12.3977 29.4463 12.0638 29.1816 11.7025 29.1816C10.806 29.1816 10.0792 30.8083 10.0792 31.7047C10.0792 32.6013 10.8061 33.3279 11.7025 33.3279C12.0644 33.3279 12.3975 33.208 12.6674 33.0078C12.2686 32.7123 12.009 32.2396 12.009 31.7049Z"
+        fill="#660C00"
+      />
+      <path
+        d="M17.305 39.5801C17.305 40.4766 16.5782 41.2033 15.6818 41.2033C14.7853 41.2033 14.0586 40.4765 14.0586 39.5801C14.0586 38.6836 14.7853 37.057 15.6818 37.057C16.5782 37.057 17.305 38.6836 17.305 39.5801Z"
+        fill="#660C00"
+      />
+      <path
+        d="M15.9881 39.5801C15.9881 39.0451 16.2471 38.25 16.6465 37.6957C16.3768 37.3215 16.0429 37.0568 15.6816 37.0568C14.7851 37.0568 14.0583 38.6836 14.0583 39.5799C14.0583 40.4765 14.7852 41.2031 15.6816 41.2031C16.0435 41.2031 16.3766 41.0832 16.6465 40.883C16.2478 40.5875 15.9881 40.1148 15.9881 39.5801Z"
+        fill="#660C00"
+      />
+    </svg>
+    Fruits
```

Comment:
Maybe you can add a span tag to the word :)

Reply:
thx I added span